pkg/config/config.go

```diff
@@ -154,6 +154,12 @@ type ContainersConfig struct {
 	// Deprecated: Do not use this field directly use conf.FindInitBinary() instead.
 	InitPath string `toml:"init_path,omitempty"`
 
+	// InterfaceName tells container runtimes how to set interface names
+	// inside containers.
+	// The only valid value at the moment is "parent" that indicates the interface
+	// name should be set to the parent interface name of the respective bridge.
+	InterfaceName string `toml:"interface_name,omitempty"`
```

Comment:
I don't like this name, it is not clear what it actually does. I would assume it sets the specific name not an option such as "parent".

IMO a global option seems wrong anyway. As you said docker uses a network specific option so shouldn't we do the same?

Reply:
@Luap99 ,

The change is based on the discussion and alignment that happened in https://github.com/containers/podman/issues/13174.

If you prefer a different attribute name and value, do suggest.

Reply:
Have changed the attribute value to "host" based on the comment https://github.com/containers/common/blob/main/libnetwork/types/network.go#L55
`// NetworkInterface is the network interface name on the host.`